tests/test_config_files.py

```diff
@@ -19,4 +19,17 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-from .version import *  # Generated by sconsUtils
+import pathlib
+import unittest
+
+from lsst.ts import salobj
+
+
+class ConfigTestCase(salobj.BaseConfigTestCase, unittest.TestCase):
+    def test_OCPS(self):
+        self.check_standard_config_files(
+            sal_name="OCPS",
+            module_name="lsst.dm.OCPS",
+            schema_name="CONFIG_SCHEMA",
```

Comment:
Thanks for the update. I assume this passes?

Reply:
Yes, I ran it and it passes.